data/deprecated.json

```diff
@@ -1205,6 +1205,126 @@
     "old": {"parking": "street"},
     "replace": {"parking": "lane"}
   },
+  {
+    "old": {"parking:lane:both": "yes"},
+    "replace": {"parking:both": "lane"}
+  },
+  {
+    "old": {"parking:lane:both": "no"},
+    "replace": {"parking:both": "no"}
+  },
+  {
+    "old": {"parking:lane:both": "no_parking"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_parking"}
+  },
+  {
+    "old": {"parking:lane:both": "no_standing"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_standing"}
+  },
+  {
+    "old": {"parking:lane:both": "no_stopping"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping"}
+  },
+  {
+    "old": {"parking:lane:both": "fire_lane"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping", "parking:both:restriction:reason": "fire_lane"}
```

Comment:
Side node: We do not have a preset for the `*:reason` tag, so this will become "invisible" to users that do not look at the "Tags" area. That is OK IMO. Its just a +1 to some day have a even more advanced preset system that would allow to have something like the reason more visible.